### ISSUE TITLE
Old Sidebar 1.1.3 disabling Messaging Bubble

### ISSUE THREAD

**Issue by Wolvan** [new-xkit/XKit#976]
For whatever reason, the CSS update in #975 made the messaging icon's menu completely unavailable. Downgrading to a prior version of the CSS fixes the issue

**Comment by hobinjk** [comment]
Can't reproduce locally, are you talking about the menu that appears when you click on the icon?

**Comment by Wolvan** [comment]
YeaH

**Comment by hobinjk** [comment]
Not seeing that on either Chrome or Firefox. Can you be more specific about the steps to reproduce?

**Comment by ValensHawke** [comment]
This just started happening to me as well. It actually is not "completely unavailable." If you click in the area where the chat dropdown is supposed to be, it'll bring up the chat that's there. The dropdown after clicking the Chat Icon in the header does not appear because of the Old Sidebar. Disabling Old Sidebar, and the dropdown of chats does appear. I disabled XKit, Chat Icon worked, then I reenabled every extension 1-3 at a time and discovered it this way. This is the only extension, when on, that's causing this. It's weird because it was working fine for a few hours after the update. 

This is for Chrome, I haven't tested on Firefox.
